pages/deploy-service-broker-locally.md

```diff
@@ -0,0 +1,195 @@
+# Table of Contents
+
+1. [Open Service Broker Framework (by evoila)](../README.md)
+2. [Components](components.md)
+3. [Configuring a Service Broker](configure-service-broker.md)
+4. [Service Keys](service-keys.md)
+5. [Backup Agent](backup-agent.md)
+6. [Development](development.md)
+7. [Local Deployment](#how-to-depoly-a-service-broker-on-your-local-machine-with-bosh-lite)
+    * 7.1 [Prerequisite](#prerequisite)
+    * 7.2 [Bosh-lite config and service broker connection](#bosh-lite-config-and-service-broker-connection)
+      * 7.2.1 [Connect the service-broker to bosh-lite](#connect-the-service-broker-to-bosh-lite)
+      * 7.2.2 [Pepare bosh and service broker manifest](#pepare-bosh-and-service-broker.manifest)
+    * 7.3 [Setup Security Components](#setup-security-components)
+      * 7.3.1 [Bosh-lite Uaa setup](#bosh-lite-uaa-setup)
+      * 7.3.2 [Connect to bosh-lite credhub](#connect-to-bosh-lite-credhub)
+    * 7.3 [Creds.yml fields](#creds.yml-fields)
+8. [Open Service Broker API Features](osb-api-features.md)
+9. [IDE & Runtime](ide-runtime.md)
+10. [Contribution](contribution.md)
+11. [License](license.md)
+---
+
+# How to depoly a service broker on your local machine with bosh-lite
+This tutorial describes how to set up a service broker locally with bosh-lite. It is focused on deploying a already created project based on osb-core.
+ After following these steps you should be able to use your service broker to deploy a service on bosh-lite, mange it with the service-broker dashboard, safe credentials in Credhub and use Uaa to authenticate users. Uaa and credhub are being deployed as a part of Bosh-lite.
+
+## Prerequisite
+Before deploying your service broker you need a running bosh-lite, a mongoDB instance and the bosh release you wish to deploy. Note your service broker may needs to implement some feature to handle deployments correctly. If you don't have deployed bosh-lite yet, refer to the bosh documentation for instructions. For a mongoDB deployment I recommend a docker container. We are hosting a docker compose file on github, that contains everything you possibly need. Refere [here](configure-service-broker.md#basic-configuration)
```

Comment:
"We are hosting a docker compose file on github, that contains everything you possibly need."
We do? Where and how's the visibility?

Reply:
It's in bit bucket. I'll reference a public MongoDb docker container.